slack-base/src/main/java/com/hubspot/slack/client/models/interaction/BlockActionsIF.java

```diff
@@ -26,6 +26,8 @@
   @JsonProperty("actions")
   List<BlockElementAction> getElementActions();
 
+  ViewStateValuesPayload getState();
```

Comment:
Ideally, we'd like to rename this class to something more generic as it doesn't represent solely "View" state anymore. The only drawback is that this is going to be a breaking change. Thoughts?

Reply:
Fixed